dev-python/qtconsole/qtconsole-4.1.1.ebuild

```diff
@@ -29,6 +29,10 @@ DEPEND="${RDEPEND}
 	)
 	test? (
 		>=dev-python/nose-0.10.1[${PYTHON_USEDEP}]
+		|| (
+			dev-python/PyQt4[${PYTHON_USEDEP},svg,testlib]
+			dev-python/PyQt5[${PYTHON_USEDEP},svg,testlib]
```

Comment:
I agree in principle, but since the bug was reported using Qt4, I would like to avoid dropping features from this version. With the next version bump we could than drop Qt4 support.